src/catalog.rs

```diff
@@ -3,77 +3,75 @@ use crate::result::{illegal_operation, illegal_operation_raw};
 use crate::shared_symbol_table::SharedSymbolTable;
 use std::collections::{BTreeMap, HashMap};
 
-/// Catalog is a collection of Shared Symbol Tables
-/// for more information on [Catalog]: https://amzn.github.io/ion-docs/docs/symbols.html#the-catalog
+/// A Catalog is a collection of Shared Symbol Tables.
+/// For more information about the concept of a catalog,
+/// see [the `symbols` section of the specification](https://amzn.github.io/ion-docs/docs/symbols.html#the-catalog).
 pub trait Catalog {
     /// Returns the Shared Symbol Table with given table name
     /// If a table with the given name doesn't exists or if the table name is an empty string
     /// then returns an error
+    /// If a table with multiple versions exists for the given name then it will return the latest version of table
```

Comment:
I think the spec says that if no version is specified, `1` is assumed. No `TODO` here, just noting that this behavior is different from the use case in the spec.

I think that for our initial implementation, we can have `ion-rust` require an exact version match. The world of partial-matching on imports is pretty complex.